src/geography.cpp

```diff
@@ -262,17 +283,23 @@ void init_geography(py::module &m) {
     // Geography types
 
     auto pygeography_types = py::enum_<GeographyType>(m, "GeographyType", py::arithmetic(), R"pbdoc(
-        The enumeration of Geography types
+        The enumeration of Geography types.
```

Comment:
Not entirely sure why this happens, but for some reason this first line is "quoted" in the docstring, as if sphinx thinks it is indented (https://spherely--89.org.readthedocs.build/en/89/_api_generated/spherely.GeographyType.html)